platform/osx/export/export.cpp

```diff
@@ -95,8 +95,8 @@ class EditorExportPlatformOSX : public EditorExportPlatform {
 	virtual void get_export_options(List<ExportOption> *r_options) override;
 
 public:
-	virtual String get_name() const override { return "Mac OSX"; }
-	virtual String get_os_name() const override { return "OSX"; }
+	virtual String get_name() const override { return "macOS"; }
```

Comment:
This is a breaking change, as many people rely on `OS.get_name()` in scripts. Moreover, this will also affect [feature tags](https://docs.godotengine.org/en/latest/getting_started/workflow/export/feature_tags.html).